types/office-js-preview/index.d.ts

```diff
@@ -69890,7 +71139,7 @@ declare namespace Excel {
              */
             category?: string;
             /**
-             * The Comments field in the metadata of the workbook. These have no connection to comments by users made in the workbork.
+             * The comment field in the metadata of the workbook. These have no connection to comments by users made in the workbork.
```

Comment:
This line and its duplications have been reverted. The source change was made on 10/14, so it's possible this is overwriting a local change and the fix is coming soon.

Reply:
Ok, thanks! No, the 10/14 changes are not in the source yet.